tests/jax/test_data_types.py

```diff
@@ -5,17 +5,20 @@
 # RUN: %PYTHON %s | FileCheck %s
 
 
-import jax
 import jax.numpy as jnp
 
 # Currently, tt::runtime only support float32, bfloat16, uint16, and uint32
 
 
+# TODO this test needs to be rewritten, this is hard to figure out.
```

Comment:
Change to "TODO Investigate the purpose of this test and if we can test it in a different way."